packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -925,6 +925,29 @@ const String kIosArchs = 'IosArchs';
 /// Supported values are x86_64 and arm64.
 const String kDarwinArchs = 'DarwinArchs';
 
+/// The define to control what Android architectures are built for.
+///
+/// This is expected to be a space-delimited list of architectures.
+const String kAndroidArchs = 'AndroidArchs';
+
+/// If the current build is `flutter build aar`.
+///
+/// This is expected to be a boolean.
+///
+/// If not provided, defaults to false.
+const String kIsAddToApp = 'IsAddToApp';
```

Comment:
This might be better named. IsAndroidLibrary or another name that indicates it is an aar build. Add to app covers more cases than this including building modules which I think you explicitly support.